docs/form_helper.rst

```diff
@@ -159,23 +165,17 @@ All previous, ``bootstrap`` (version 2 and 3) attributes are also settable in bo
 .. |customfile| image:: images/custom_file_field.png
    :width: 300px
    :align: middle
-.. |clearablefile| image:: images/clearable_file.png    
```

Comment:
Dropping clearable file from here is an unrelated change?

Reply:
Yes (I noted this in the commit, perhaps I should move it to a PR): https://github.com/cpina/django-crispy-forms/commit/c1340236525c5712f05184488ccc45580f77a8d9

The file field custom clearable png is, at the moment, broken in the documentation: https://django-crispy-forms.readthedocs.io/en/latest/form_helper.html?bootstrap-4-helper-attributes#bootstrap-4-helper-attributes

Reply:
> Dropping clearable file from here is an unrelated change?

I've created a branch minimising this change: https://github.com/cpina/django-crispy-forms/tree/custom-select-with-minimal-docs-change . I might have got a bit carried on the other day. I can make the other changes (delete the clearable one maybe, re-order the table so it matches the text) in another PR.

Reply:
The clearable file also looks awful on rtd, I'm not sure it's that helpful.